Detections/MultipleDataSources/ExchangeServerVulnerabilitiesMarch2021IoCs.yaml

```diff
@@ -112,22 +135,6 @@ query: |
   | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated) by SourceIP, DestinationIP, DeviceProduct, DeviceAction, Message, Protocol, SourcePort, DestinationPort, DeviceAddress, DeviceName, IPMatch
   | extend timestamp = StartTimeUtc, IPCustomEntity = case(IPMatch == "SourceIP", SourceIP, IPMatch == "DestinationIP", DestinationIP, "IP in Message Field")
   ),
-  (VMConnection
-  | where isnotempty(SourceIp) or isnotempty(DestinationIp)
-  | where SourceIp in (ips) or DestinationIp in (ips)
-  | extend IPMatch = case( SourceIp in (ips), "SourceIP", DestinationIp in (ips), "DestinationIP", "None")
-  | extend timestamp = TimeGenerated , IPCustomEntity = case(IPMatch == "SourceIP", SourceIp, IPMatch == "DestinationIP", DestinationIp, "None"), Host = Computer
-  ),
-  (Event
```

Comment:
I was wrong, we don't do Sysmon event 3 for Windows today :-(. We really should.